README.md

```diff
@@ -138,12 +138,35 @@ Noindex: other-example.html
 
 #### Canonical URL Tag
 
-The SEO Enhancer automatically adds canonical link metadata to `<head>` tags in order to avoid presenting duplicate content to search engines.
+The SEO Enhancer automatically adds canonical link metadata to `<head>` tags in order to avoid presenting duplicate content to search engines:
+
+1. If you have declared the `:external_canonical:` metadata field, the SEO enhancer will use this. It expects a URL, e.g. https://www.example.com/external_canonical_article.html. It can be used if you syndicate content from third parties or want to include your own posts that are already hosted elsewhere. The SEO enhancer will prefer `:external_canonical:` over the `:save_as:` metadata field.
```

Comment:
```suggestion
1. If you have declared the `:external_canonical:` metadata field, the SEO Enhancer will use it. It expects a URL, e.g. `https://www.example.com/external_canonical_article.html`. It can be used if you syndicate content from third parties or want to include your own posts that are already hosted elsewhere. The SEO Enhancer will prefer `:external_canonical:` over the `:save_as:` metadata field.
```